ppa/archive/urls.py

```diff
@@ -1,13 +1,20 @@
 from django.conf.urls import url
+from django.contrib.admin.views.decorators import staff_member_required
 
 from ppa.archive import views
 
 
 urlpatterns = [
     url('^collections/$', views.CollectionListView.as_view(),
         name='list-collections'),
+    url(
+        '^collections/add/$',
+        staff_member_required(views.BulkAddCollectionView.as_view()),
+        name='bulk-add'
```

Comment:
I think this url name isn't specific enough and might get confusing as we add more views.  Could you revise to include collection somehow?  Maybe bulk is unnecessary, since we're unlikely to have other add-to-collection views.

The url isn't very resource-y, but I'm having trouble coming up with anything better.

Reply:
Do you have any suggestions for this url terminology or the view name, @rlskoeser, having had some time to mull it?  If not, I'll think and take another stab at it

Reply:
What do you think of `add-to-collection` for url name, and `AddToCollection` for the view?

Reply:
I like it. I'll change the url and view accordingly. Thank you!